pkg/kubelet/kubelet_node_status.go

```diff
@@ -572,6 +571,19 @@ func (kl *Kubelet) setNodeStatusMachineInfo(node *v1.Node) {
 				}
 			}
 		}
+
+		if utilfeature.DefaultFeatureGate.Enabled(features.DevicePlugins) {
```

Comment:
It is probably safe to skip this feature checking anyway, but I am following the current model as LocalStorageCapacityIsolation.